build_tools/azure/install_pyodide.sh

```diff
@@ -14,9 +14,3 @@ pip install pyodide-build==$PYODIDE_VERSION pyodide-cli
 pyodide build
```

Comment:
Would it be possible to speed-up the build by configuring ccache as we do for the other linux and macos builds?

Reply:
Maybe in a separate PR?

I tried setting ccache with emscripten locally and for some reason, I get a poor cache hit rate of ~30%. Locally building without the cache takes ~4 minutes and with the cache ~3 minutes. I am not too sure how this would translate in the CI.